.github/renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs",
+    ":enableVulnerabilityAlertsWithLabel('security')"
+  ],
+  "ignorePresets": [
+    ":prHourlyLimit2"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "io.quarkiverse.cxf"
+      ],
+      "enabled": false
+    }
+  ]
```

Comment:
There is an ignore rule in the dependabot file: https://github.com/quarkiverse/quarkus-cxf/blob/main/.github/dependabot.yml#L13
Not sure it still applies after the switch to Quarkus 3?

Reply:
Yes, I don't believe the exclusion applies any longer given our Quarkus 3 / Jakarta updates.